components/ui/Headings.jsx

```diff
@@ -0,0 +1,35 @@
+export const H1 = ({ text, styles }) => {
+    return (
+        <h1 className={`lg:text-23xl lg:leading-15.332  ${styles}`}>{text}</h1>
```

Comment:
in the typography, the breakpoint of the tablets are not considered , include them too

Reply:
As on the design, tablet size is 1024px, which is lg on tailwind responsive design. Hence we are considering lg because values such as font size and leading on desktop are equal to the values on tablet. Thus, applying them on lg(tablet) will also take effect on xl(desktop)